rooms/templates/rooms/rooms_list.html

```diff
@@ -13,7 +13,7 @@ <h1>Rooms List</h1>
 
 <ul>
     {% for room in object_list %}
-        <li> {{room}}</li>
+        <li><a href="/{{room.pk}}/details">{{room}}</a></li>
```

Comment:
_nice_ job including the link, but we should use Django's `url` tag here instead of a relative link: https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#url